hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/service/DeduplicationAwaitingService.java

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.securecodebox.persistence.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+public class DeduplicationAwaitingService {
+    private static final Logger LOG = LoggerFactory.getLogger(DeduplicationAwaitingService.class);
+
+    Integer refetchWaitSeconds;
```

Comment:
For what reason is this a object property, since it is only used in the method `#refetchWait()`?

Reply:
@Weltraumschaf I should probably have added a comment that I am by no means a Java developer ^^
I am happy, that I got it working at all.

What exactly should I change here?

Reply:
No problem, I jump in, i've some experience in Java :-)